rclpy/package.xml

```diff
@@ -14,9 +14,13 @@
   <depend>rmw_implementation</depend>
   <depend>rcl</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
+
   <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_index_python</test_depend>
```

Comment:
So, after removing this I get this failure:

http://ci.ros2.org/job/ci_linux/1127/testReport/junit/(root)/projectroot/rclpytests/

I think without the `test_depend` it doesn't work. Maybe it's a bug in `ament`, but I thought I remembered needing to add this explicitly, but couldn't remember if it was for this dependency or for `launch`.